sdk/core/az_core/inc/az_log.h

```diff
@@ -55,6 +55,9 @@ typedef enum
 
   AZ_LOG_IOT_SAS_TOKEN
   = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT, 2), ///< IoT Client generated new SAS token.
+
+  AZ_LOG_IOT_AZURERTOS
```

Comment:
nit: Should this have an underscore between Azure and RTOS?
```suggestion
  AZ_LOG_IOT_AZURE_RTOS
```

Reply:
Rick and I decided against the additional underscore. We tried it and it made it look like azure and rtos were not related to each other but this is specifically for "azure rtos"